MUDscape/driver.py

```diff
@@ -25,7 +25,11 @@ def handle(self):
         self.loop()
 
     def loop(self):
+        #KNOWN BUG: if not connecting from something like putty, relevant information is stored in junk
+        #TODO need to distinguish between systems and determine whether to throw away input or not.
+        junk = self.request.recv(1024)
```

Comment:
You should parse this to see if it starts with an `IAC` character, `0xFF`, then determine if it's to be ignored.